pkg/utils/utils.go

```diff
@@ -788,6 +788,22 @@ func RunCommand(command string, args ...string) (string, error) {
 	return stdout.String(), err
 }
 
+func RunOVSvsctl(args ...string) (string, error) {
+	glog.Infof("runOVSvsctl(): ovs-vsctl %v", args)
+	var output bytes.Buffer
+	cmd := exec.Command("ovs-vsctl", args...)
+	cmd.Stdout = &output
+	err := cmd.Run()
+
+	return output.String(), err
+}
+
+func RestartOVS() error {
+	glog.Infof("RestartOVS(): restarting openvswitch.service")
+	cmd := exec.Command("systemctl", "restart", "openvswitch.service")
```

Comment:
did you test it ith Ubuntu 22.04? AFAIK, service name in Ubuntu is different

Reply:
Actually, this is a good point. Let's also use the systemd package in go to start services

Reply:
Actually, we have to cordon berfore and uncordon after like 803.